src/integrationTest/java/uk/gov/hmcts/darts/cases/service/CaseServiceAdvancedSearchUseInterpreterTest.java

```diff
@@ -271,7 +271,7 @@ void testSearchCasesWithPermissionsGlobalTrueOnCourthouselessUserGroupUseInterpr
 
         // then
         var caseNumbers = resultList.stream().map(AdvancedSearchResult::getCaseNumber).toList();
-        assertThat(caseNumbers).containsExactlyInAnyOrder("Case1", "Case2", "Case4", "Case6");
+        assertThat(caseNumbers).containsExactlyInAnyOrder("Case1", "Case2", "Case3", "Case4", "Case5", "Case6");
```

Comment:
I'm puzzled by this result. 
If we have set in line 263 use interpreter `true`, how is it possible that the user can access Swansea `Case1` that does not have an interpreter?

Reply:
this might be a no longer valid test case now, since any user with translation QA + interpreter flag should not have any other role. Should we keep it or remove it?
Same consideration for some other test cases in this class.

Reply:
agree, a translation QA role cant have any other role.